<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Aswinikumars-MacBook-Pro.local" name="Default test" tests="37" failures="5" timestamp="27 May 2020 18:02:33 GMT" time="1014.811" errors="0">
  <testcase name="C333216_Verify_cust_param_cnd" time="17.511" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C344257_Verify_Video_Url_Parameter_cmsid" time="0.57" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C344258_Verify_Video_Url_Parameter_ttid" time="0.633" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C344260_Verify_Video_Url_Parameter_lnid" time="0.729" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C344259_Verify_Video_Url_Parameter_vid" time="0.622" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C344261_Verify_Video_Url_Parameter_iu" time="0.648" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C344261_Verify_DescriptionURL_Video" time="0.488" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C344261_Verify_ContentURL_Video" time="0.458" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C658716_Verify_cust_param_tf" time="8.046" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333213_Verify_cust_param_adid" time="10.918" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333213_Verify_cust_param_aid" time="10.91" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333219_Verify_cust_param_st" time="10.949" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333200_Verify_cust_param_ord" time="10.981" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333204_Verify_cust_param_rmid" time="11.252" classname="twc.Regression.TestCases.regressionTestCases">
    <failure type="java.lang.AssertionError" message="rmid Value not matched">
      <![CDATA[java.lang.AssertionError: rmid Value not matched
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Regression.TestCases.Custom_Parameters_Verification.parameters_Verification(Custom_Parameters_Verification.java:585)
	at twc.Regression.TestCases.regressionTestCases.C333204_Verify_cust_param_rmid(regressionTestCases.java:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- C333204_Verify_cust_param_rmid -->
  <testcase name="C333219_Verify_cust_param_ver" time="11.658" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C658711_Verify_cust_param_lang" time="12.306" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C658712_Verify_cust_param_plat" time="14.688" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333218_Verify_cust_param_pos" time="17.453" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333205_Verify_cust_param_tile" time="21.423" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333212_Verify_cust_param_ct" time="25.623" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333203_Verify_cust_param_dma" time="25.383" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C628160_Verify_Parameter_DyNght" time="20.851" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333209_Verify_cust_param_cc" time="19.802" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333232_Verify_cust_param_fhic" time="23.409" classname="twc.Regression.TestCases.regressionTestCases">
    <failure type="java.lang.AssertionError" message="fhic Value not matched">
      <![CDATA[java.lang.AssertionError: fhic Value not matched
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Regression.TestCases.Custom_Parameters_Verification.parameters_Verification(Custom_Parameters_Verification.java:631)
	at twc.Regression.TestCases.regressionTestCases.C333232_Verify_cust_param_fhic(regressionTestCases.java:220)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- C333232_Verify_cust_param_fhic -->
  <testcase name="C333238_Verify_cust_param_floc" time="22.442" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333208_Verify_cust_param_tmp" time="28.671" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333217_Verify_cust_param_tmpr" time="40.5" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333239_Verify_cust_param_tmpc" time="30.513" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333202_Verify_cust_param_zip" time="22.072" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333207_Verify_cust_param_hmid" time="25.672" classname="twc.Regression.TestCases.regressionTestCases">
    <failure type="java.lang.AssertionError" message="hmid Value not matched">
      <![CDATA[java.lang.AssertionError: hmid Value not matched
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Regression.TestCases.Custom_Parameters_Verification.parameters_Verification(Custom_Parameters_Verification.java:211)
	at twc.Regression.TestCases.regressionTestCases.C333207_Verify_cust_param_hmid(regressionTestCases.java:585)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- C333207_Verify_cust_param_hmid -->
  <testcase name="C333221_Verify_cust_param_wind" time="26.531" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333224_Verify_cust_param_uv" time="61.918" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333232_Verify_cust_param_fltmpc" time="59.886" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333228_Verify_cust_param_wfxtg" time="17.965" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333214_Verify_cust_param_hlzip" time="389.334" classname="twc.Regression.TestCases.regressionTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at twc.Regression.TestCases.Custom_Parameters_Verification.parameters_Verification(Custom_Parameters_Verification.java:390)
	at twc.Regression.TestCases.regressionTestCases.C333214_Verify_cust_param_hlzip(regressionTestCases.java:286)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- C333214_Verify_cust_param_hlzip -->
  <testcase name="C333211_Verify_cust_param_locale" time="11.469" classname="twc.Regression.TestCases.regressionTestCases"/>
  <testcase name="C333223_Verify_cust_param_sev" time="0.43" classname="twc.Regression.TestCases.regressionTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at twc.Regression.CustomParamValidation.CustomParamFunctions.validate_results(CustomParamFunctions.java:46)
	at twc.Regression.TestCases.Custom_Parameters_Verification.parameters_Verification(Custom_Parameters_Verification.java:646)
	at twc.Regression.TestCases.regressionTestCases.C333223_Verify_cust_param_sev(regressionTestCases.java:360)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- C333223_Verify_cust_param_sev -->
</testsuite> <!-- Default test -->
